collab-folder/src/core/view.rs

```diff
@@ -64,6 +64,46 @@ impl ViewsMap {
     self.remove_cache_view(parent_id);
   }
 
+  /// why we did not use `move_child`?
+  /// because we need to move child view from one parent to another parent
+  /// but the new parent maybe not view, like move out from view to workspace
+  pub fn move_child_out(&self, parent_id: &str, view_id: &str) {
+    self.container.with_transact_mut(|txn| {
+      self.move_child_out_with_txn(txn, parent_id, view_id);
+    })
+  }
+
+  /// why we did not use `move_child`?
```

Comment:
Use tool to correct your sentence: 

/// Why didn't we use move_child?
/// It's because we need to move the child view from one parent to another.
/// However, the previous parent might not be a view, such as when moving a view from the workspace.